### PR TITLE
fixed bug for PhyloPhlAn wrapper

### PR DESCRIPTION
**Pull request by qiyunzhu**
Just a simple fix.

### CODE REVIEW

**Comment by mortonjt** [comment]
looks good.

File: horizomer/benchmark/run_phylophlan.sh [hunk 1]
```diff
@@ -11,22 +11,21 @@
...
 
 # prerequisites:
-# one needs to install the development version of PhyloPhlAn:
-#   hg clone https://bitbucket.org/nsegata/phylophlan
-#   cd phylophlan
-#   hg up dev
+# one needs to install a certain development version of PhyloPhlAn:
+#   hg clone https://bitbucket.org/nsegata/phylophlan -r 2c0e61a
+# one needs a Python 2 conda environment with biopython installed
 # one needs the follow programs installed and callable from $PATH:
```
Comment (anupriyatripathi) [inline horizomer/benchmark/run_phylophlan.sh:17]: following* programs

File: horizomer/benchmark/run_phylophlan.sh [hunk 2]
```diff
@@ -11,22 +11,21 @@
 # usage: run PhyloPhlAn to build phylogenetic tree of whole genomes
 
 # prerequisites:
-# one needs to install the development version of PhyloPhlAn:
-#   hg clone https://bitbucket.org/nsegata/phylophlan
-#   cd phylophlan
-#   hg up dev
+# one needs to install a certain development version of PhyloPhlAn:
+#   hg clone https://bitbucket.org/nsegata/phylophlan -r 2c0e61a
+# one needs a Python 2 conda environment with biopython installed
```
Comment (anupriyatripathi) [inline horizomer/benchmark/run_phylophlan.sh:16]: am I missing the command to create the mentioned conda environment?

File: horizomer/benchmark/run_phylophlan.sh [hunk 3]
```diff
@@ -48,14 +45,11 @@ mkdir -p input
...
-ln -s ${input_faa_dir} input/genomes
+ln -s ${input_faa_dir} input/all
 
 # command
-cmd="${phylophlan_install_dir}/phylophlan.py -u input --nproc $threads --c_dat temp"
-if [ "$verbose" == "true" ]
-then
-    echo "Command:"$'\n'"  $cmd"
-fi
+cmd="${phylophlan_install_dir}/phylophlan.py -u all --nproc $threads --c_dat temp"
```
Comment (anupriyatripathi) [inline horizomer/benchmark/run_phylophlan.sh:51]: -u input/all ?

**Review by anupriyatripathi — COMMENTED** [review]
Minor doubts. Maybe you're all good and I am missing something!

**Comment by qiyunzhu** [comment]
Hi @anupriyatripathi Thanks for reviewing! Jamie already merged the PR. Please find my responses to your comments.

File: horizomer/benchmark/run_phylophlan.sh (hunk 2 above)
Comment (qiyunzhu) [reply horizomer/benchmark/run_phylophlan.sh:16]: PhyloPhlAn wasn't designed to run in a conda environment. (It was designed before conda became popular.)

File: horizomer/benchmark/run_phylophlan.sh (hunk 1 above)
Comment (qiyunzhu) [reply horizomer/benchmark/run_phylophlan.sh:17]: You are right! Will change in next PR.

File: horizomer/benchmark/run_phylophlan.sh (hunk 3 above)
Comment (qiyunzhu) [reply horizomer/benchmark/run_phylophlan.sh:51]: This is but a naming preference. Here `all` or `input` is to describe the job. It is totally arbitrary.

**Pull request merged**